webapp/src/app/models/Material.ts

```
@@ -20,4 +21,5 @@
   //userID:string;
   channelId:string;
   description:string
-}
+  showDialog?:Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)